packages/mobile/src/qrcode/utils.ts

```diff
@@ -64,6 +69,35 @@ export function* handleBarcode(
     yield put(showError(ErrorMessages.QR_FAILED_INVALID_ADDRESS))
     return
   }
+  try {
```

Comment:
Nit: would be nice if this stuff was in a separate function if possible

Reply:
Good suggestion. Done.